src/libmw/defs.cpp

```diff
@@ -52,6 +52,21 @@ MWEXPORT std::vector<PegOut> TxRef::GetPegouts() const noexcept
     return pegouts;
 }
 
+MWEXPORT std::vector<PegIn> TxRef::GetPegins() const noexcept
+{
+    std::vector<PegIn> pegins;
+    for (const Kernel& kernel : pTransaction->GetKernels()) {
+        if (kernel.IsPegIn()) {
+            PegIn pegin;
+            pegin.amount = kernel.GetPeggedOut();
```

Comment:
Should be GetPeggedIn.

Reply:
Oops! Fixed locally. I'll include that in a future PR.